OracleOGG/BigData/12.3.0.1.0/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM oraclelinux:7-slim
+MAINTAINER OGG QA <hadi.koesnodihardjo@oracle.com>
```

Comment:
If you're using a user-specific email address, please provide the user's name. If you want to specify a group, please use a mailing list or email alias instead.